README.md

```diff
@@ -1,6 +1,6 @@
 # Deobfuscator
  
-This project aims to deobfuscate most commercially-available obfuscators for Java.
+Java-Deobfuscator is a program that deobfuscates obfuscated Java bytecodes, makes it much clear and readable (and easier to decompile). This project aims to deobfuscate most commercially-available obfuscators for Java.
```

Comment:
While the purpose of this is to clarify what java-deobfuscator does, I feel this addition is unnecessary because the exact same thing is restated in the very next sentence. I think we can probably just stick with the original description.

Reply:
At first, i though that these sentences are looking pretty similar but first sentence explains "what it does" and second one explains "what it is meant to be".  Most of the people just see README.md and i think it is important to identify what this project is about. If you still think it's unnecessary, tell me why. (My main language is not English)

Reply:
I believe it's unnecessary because the word "deobfuscation" implies the purpose of the project is to clean up obfuscated bytecode, and the first sentence can actually be inferred by the second (and vice versa). Having one or other works well (though, I would personally reword the first one) but not quite as well when both are present.

Reply:
As my english is not good enough to explain such thing, I replaced the original one back. I am not even sure the first one is good or not, but I couldn't write any better sentence. Deobfuscator doesn't deobfuscate obfuscators, it reverts what they done to bytecode, it deobfuscates bytecode. 

I won't edit this anymore but i think original description could be more descriptive.

Reply:
That's a good point. I've added a new note to rewrite the description for clarity sometime in the future.